### ISSUE TITLE
431 Error

### ISSUE THREAD

**Issue by ebraker** [ArctosDB/arctos#6415]
I'm getting a 431 error when:

- trying to select an existing profile for search results
- trying to create a new profile for search results
- clicking "request data"
![image](https://github.com/ArctosDB/arctos/assets/11336485/76df3646-a81d-44dd-b114-03992ee5bb87)

help?

**Comment by dustymc** [comment]
Need more info - what's the URL? That error is usually from a too-long URL, but none of the things you describe should lead there. Maybe some add-on going crazy??

**Comment by ebraker** [comment]
I have a long URL ;) Despite the error pages it looks like my test profiles saved. Now I'm no longer able to recreate the error...

https://arctos.database.museum/search.cfm?guid_prefix=UCM%3AHerp&cat_num=63741%2063742%2064369%2063944%2011982%2063745%2064368%2011976%2011979%2064367%2063737%2048151%207547%2032306%2011980%2016953%2048147%207548%2011975%2048148%2032311%2016954%2032308%2011978%207545%2048149%2048150%2048146%2032307%2067294%2032305%2011981%2032316%2032310%2063738%2032313%2067295%2032314%2011977%2032312%2032309%2032315%2068353%2028429%2028414%2028441%2028434%2028447%2028427%2028412%2028410%2028416%2028415%2028448%2028430%2028418%2028423%2028438%2028417%2018602%2018604%2018600%2066395%2066955%2066396%2066948%2066393%2066401%2066398%2066392%2066400%2066450%2066402%2066448%2066476%2066397%2066943%2066425%2066399%2066422%2066945%2066403%2066394%2066414%2066418%2061188%2066406%2066408%2066409%2066407%2066405%2066413%2066410%2066412%2066417%2066424%2066473%2066421%2066420%2066474%2066416%2066411%2066483%2066475%2066415%2066419%2066447%2020054%2014647%2020074%2020061%2020077%2020060%2020057%2056596%2014648%2016942%2013620%2020056%2029550%2020062%2059300%2016940%2014645%2029552%2020072%2020058%2020079%2020064%2016939%2020078%2013621%2014649%2014646%2029549%2016941%2020065%2020069%2056595%2029551%2027271%2020076%2024226%2029554%2029548%2020070%2024225%2029781%2020073%2029553%2016938%2012159%2012153%2012161%2012123%2012146%2012119%2012144%2012160%2012166%2012120%2012139%2012164%2012132%2012154%2012130%2012137%2012143%2012127%2012156%2012138%2012152%2012158%2012121%2012124%2012128%2012134%2012167%2012133%2012162%2012126%2012117%2012110%2012165%2012142%2012125%2012141%2012168%2012122%2012135%2012163%2012148%2012157%2012118%2012131%2012151%2012111%2012149%2012109%2012147%2012112%2012108%2012136%2012113%2012155%2012150%2012116%2012140%2012114%2012115%2013383%207396%2013381%2013378%207394%2013379%2013391%2013389%2013394%2013393%2013387%207393%2013380%2013388%2013385%2013392%2013382%2013384%2013386%207395%207397%2013377%2068652%2068651%2068653%2068654%2068655%2068656%2068657%2068658%2068659%2068660%2068661%2068662%2068645%2068646%2068647%2068648%2068355%2068356%2068357%2068358%2068359%2068360%2068361%2068362%2068365%2068366%2068367%2068368%2068369%2068370%2068371%2068372%2068375%2068376%2068377%2068378%2068379%2068380%2068381%2068382%2068383%2068384%2068385%2068386%2068387%2068389%2068390%2068391%2068392%2068393%2068394%2068395%2068396%2068397%2068399%2068400%2068401%2068402%2068403%2068404%2068405%2068406%2068407%2068388%2068398%2068410%2068411%2068412%2068413%2068414%2068415%2068416%2068417%2068418%2068419%2068420%2068421%2068422%2068423%2068424%2068425%2068426%2068427%2068428%2068429%2068430%2068431%2068432%2068433%2068434%2068435%2068436%2068437%2068438%2068439%2068442%2068443%2068444%2068445%2068446%2068447%2068448%2068449%2068450%2068451%2068452%2068453%2068454%2068455%2068456%2068457%2068458%2068459%2068460%2068461%2068462%2068463%2068464%2068465%2068466%2068467%2068468%2068469%2068470%2068473%2068474%2068475%2068476%2068477%2068478%2068479%2068480%2068481%2068482%2068483%2068484%2068485%2068486%2068487%2068488%2068489%2068490%2068491%2068510%2068511%2068512%2068513%2068514%2068497%2068498%2068499%2068492%2068493%2068494%2068495%2068496%2068500%2068501%2068502%2068503%2068504%2068505%2068506%2068507%2068515%2068516%2068517%2068518%2068519%2068520%2068521%2068522%2068523%2068524%2068527%2068528%2068529%2068532%2068537%2068538%2068541%2068542%2068543%2068544%2068545%2068546%2068547%2068548%2068551%2068552%2068553%2068554%2068555%2068556%2068557%2068558%2068559%2068560%2068561%2068562%2068563%2068564%2068565%2068566%2068567%2068570%2068571%2068572%2068573%2068574%2068575%2068576%2068577%2068578%2068579%2068580%2068583%2068584%2068585%2068586%2068587%2068588%2068589%2068590%2068591%2068592%2068593%2068595%2068594%2068596%2068597%2068598%2068599%2068600%2068601%2068602%2068603%2068604%2068605%2068606%2068607%2068610%2068611%2068612%2068613%2068614%2068615%2068616%2068617%2068618%2068619%2068620%2068621%2068622%2068623%2068624%2068625%2068626%2068627%2068628%2068629%2068630%2068631%2068632%2068633%2068634%2068635%2068636%2068637%2068638%2068639%2068640%2068641%2068643%2060774%2061175%2061174%2058274%2058273%2063900%2017064%2032321%2032323%2032326%2020051%2032325%2032322%2032320%2032324%2051420%2031339%2045887%2051848%2021991%209729%2021992%201080%2013714%2051101%2051100%2051097%2051094%2051098%2031345%2051092%2051096%2051093%2040789%2040791%2040788%2040787%2051095%2031341%2031351%2051099%2031352%2045891%2031342%2040790%2031355%2045893%2031343%2031344%2031354%2031348%2045892%2031350%2031353%2031346%2031347%2031349%2035220%2035224%2035223%2035221%2035217%2035218%2035219%2035222%2026103%2026104%2026105%2026106%2026179%2026170%2026165%2026178%2026172%2026174%2026168%2026166%2026175%2026177%2026173%2026171%2026163%2026176%2026164%2026167%2026169%2043875%2043872%2043871%2043874%2043852%2043853%2043854%2043851%2032433%2032432%2043873%2032431%207391%207390%207387%207386%207385%207388%207389%2023231%2023235%2023230%2023233%2023229%2023238%2023232%2023237%2023234%2023236%2016386%2019191%2016381%2016383%2020497%2023251%2016382%2016388%206365%2019192%2016387%2043893%2016385%2016380%2016384%206361%206357%2043892%207598%207601%206352%206360%206351%207599%206364%206356%206363%206359%206358%206354%206355%206353%206362%206348%206350%207600%206337%206349%206345%206338%206340%206342%206346%206343%206341%206347%206344%206339%208605%2035418%2035402%2026970%2035408%2035410%2035422%2035416%2035406%2035412%2035419%2035420%2035415%2035409%2035421%2035414%2035405%2035413%2035411%2035407%2026987%2035404%2026986%2035417%2026965%2026962%2026982%2035401%2026983%2026963%2026971%2026972%2026988%2026985%2026973%2026981%2026967%2026964%2026980%2026977%2026979%2026978%2026961%2026984%2026969%2026974%2026968%2013291%2012085%2012082%2052166%2020024%2040868%2012084%2020023%2012083%2029118%2029117%2029131%2029113%2029138%2029139%2029114%2029127%2029111%2029130%2029112%2029123%2029115%2029136%2029120%2029124%2029116%2029133%2029137%2029119%2029125%2029126%2029121%2029132%2029129%2029128%2063412%2048372%2048376%2048375%2048374%2016669%2039841%2039842%2061244%2061242%2061245%2061248%2061243%2061240%2061241%2061246%2061276%2061273%2061270%2061278%2061271%2061268%2061267%2061275%2061269%2061279%2061280%2061259%2061264%2061257%2064745%2061258%2061262%2061255%2061263%2061261%2064157%2061256%2064154%2061260%2030392%2030401%2030367%2030410%2030375%2030362%2030364%2030441%2030366%2030387%2030413%2030371%2030399%2030403%2030402%2030393%2030418%2030386%2030412%2030390%2030385%2030416%2030415%2030397%2030404%2030368%2030417%2030391%2030408%2030398%2030384%2030406%2030419%2030379%2030409%2030411%2030414%2030378%2030395%2030396%2030388%2048945%2056482%2048948%2056479%2056481%2056478%2056477%2058947%2052642%2056484%2048946%2056480%2056483%208988%208984%208980%208993%208992%208990%208998%208987%208985%208989%208991%208986%208997%208995%208981%208983%208982%2016796%2063399%2016795%2045613%2045614%2016797%2016798%2020963%2028733%2020961%208961%208950%208947%208948%208949%208952%208953%208959%208954%208960%208957%208944%208958%208956%208955%208951%208946%208941%208943%208942%208940%2035013%2035017%2035002%2035020%2035016%2035015%2035014%2035019%2035003%2035018%2035006%2035022%2035010%2035004%2035012%2035001%2035021%2035009%2035008%2028837%2028832%2028835%2028851%2028834%2028838%2028853%2028839%2028854%2028845%2028848%2028852%2028855%2028861%2028850%2028844%2028840%2028847%2028831%2028841%2028857%2028856%2028843%2028849%2028842%2028859%2028833%2028863%2028862%2028860%2028846%2028858%2028791%2028798%2028784%2028795%2028796%2028792%2028775%2028794%2028785%2028800%2028788%2028787%2028789%2028779%2028783%2028786%2028790%2028773%2028776%2028797%2028781%2028774%2028780%2028799%2057100%2028808%2028827%2028804%2028826%2028823%2028805%2028818%2028813%2028803%2028801%2028812%2028824%2028825%2028820%2028829%2028814%2028806%2028830%2028821%2028828%2028822%2028815%2028802%2028816%2028807%2028809%2028819%2018981%2039664%2018982%2018979%2018980%2018975%2039669%2018974%2039667%2039668%2039840%2018978%2018973%2039663%2018976%2039665%2049671%2049670%2049674%206295%206285%206256%206293%206271%206280%206299%206284%206296%206283%206290%206279%2034145%206265%206275%206297%2034144%206260%206301%206302%206303%206270%206286%206292%206289%206254%206274%206264%206281%206255%206273%206278%206272%206300%206261%206298%206268%206287%206257%206259%206267%206263%206266%206288%2034146%206277%206258%206262%2034142%206282%2034140%2034141%2034143%207367%207382%207357%207374%207358%207361%207378%207373%207370%207360%207375%207383%207363%207384%207380%207365%207369%207359%207368%207377%207379%207364%207372%207376%2016824%2016825%2023451%2016828%2023455%2016837%2010626%2023454%2023449%2010627%2043906%2023450%2016826%2016833%2023452%2043869%2016820%2016829%2016834%2023453%2023220%2043945%2016830%2016823%2043946%2016836%2043849%2043947%2016831%2043898%2043850%2023221%2043909%2043899%2016832%2043900%2043953%2043908%2016822%2043870%2043951%2023219%2023222%2043897%2043907%2043905%2016827%2023461%2023462%2023457%2023458%2023456%2024095%2010485%2010464%2010488%2024061%2024057%2024067%2024097%2024087%2010493%2024056%2010788%2018130%203179%2021708%202731%2024098%201365%2024046%2024082%2024060%2023943%202745%2024077%2022009%2024073%2024078%2024100%2022006%2024048%2010789%2018134%2024088%2024068%2021705%2024099%2024054%202733%2018133%2024070%202742%2010490%2024050%205424%205432%205421%204192%204181%205443%205430%205419%205436%204185%204190%205423%205427%205440%205422%205426%205442%204172%204178%204182%204174%204184%204171%203717%204173%204191%205429%205420%204177%204175%205439%205428%204183%205425%205441%205431%204179%204187%204176%204188%205433%204168%205435%204186%204170%2017422%2017425%2017430%2017432%2017429%2017426%2017428%2017421%2017427%2017418%2017424%2017423%2017420%2017431%2017433%2017419%2047575%2011448%2043423%2043426%2011452%2011469%2011467%2011463%2043428%2047569%2043432%2011464%2047581%2047571%2018648%2011449%2043427%2043430%2043422%2011446%2043429%2011453%2011450%2011459%2011456%2011466%2043424%2011447%2011458%2011470%2011445%2011468%2011451%2011455%2011465%2011462%2011457%2011454%2011461%2011852%2052007%20208%204206%204246%204194%204199%2043379%2066%204203%204193%201936%2043375%204198%204197%2043378%204245%204241%2043377%2056163%2056161%2056164%204251%204210%204200%204248%2021539%204204%2018649%204195%204243%204202%204247%204207%204253%204242%204250%204244%2021540%204201%204209%204208%204205%204249%204252%204196%2058772%204648%204647%204652%204651%204649%204650%204608%204606%204612%2022058%204607%204619%204605%204620%204609%204618%204610%204613%204611%204614%204615%204617%204616%204646%204624%204631%204629%204625%204636%204642%204626%204621%204632%204641%204627%204640%204637%204638%204623%204634%204630%204622%204644%204633%204645%204639%204628%204219%204233%204229%204225%204240%204223%204230%204212%204238%204222%204232%204236%204215%204217%204228%204211%204218%204234%204226%204227%204214%204235%204224%204231%204237%204213%204221%204216%204220%204239%2056495%2056497%2056496%2056103%2056498%2056112%2056108%2056109%2056486%2056107%2056104%2056111%2056105%2056084%2056485%2056110%2056106%2056102%2056085%2056088%2056510%2056086%2056487%2056082%2056160%2056079%2056081%2056090%2056511%2056080%2056083%2056087%2056094%2056078%2056089%2056092%2056091%2056159%2057032%2057031%2057029%201267%2057033%2057035%2057030%2020419%2020424%2020420%2021764%2021767%2043217%2021769%2021768%2021763%2020426%2020423%2020417%2021766%2021771%2020418%2020421%2020425%2020422%2021765%2021982%2020431%2020429%2021770%2019679%2020427%2020430%2020428%2019680%2010562%2023441%2023444%2023447%2023442%2023445%2023448%2010559%2010561%2030963%2010560%2023443%2023217%2023446%2010563%2023218%2059567%2059572%2059566%2059569%2059573%2059570%2059571%2059574%2016963%2017443%2017415%2017456%2010832%2017441%2056976%2059568%2016966%201063%2016964%2055663%2016965%2016962%205979%205977%205981%205980%205978%204988%204983%204989%205976%204990%204986%204984%204987%202230%204985%203967%204982%205982%203966%203965%2043255%2043381%2043262%2043263%2043530%2043254%2043531%2043528%2043527%2043264%2043984%2043261%2043983%2043981%2043206%2043308%2043529%2043382%2043258%2043257%2043532%2043260%2043256%2043982%2043205%2043259%2043204%2058780%2058778%2058777%2058789%2058779%2058781%207081%205041%203003%203017%203007%203011%202997%203001%205062%203010%205106%205098%205104%202996%205040%205105%205107%205116%205115%205100%205097%205096%205103%205112%205108%205117%205039%205055%205078%207092%207086%203013%203014%202998%205211%203016%202990%205214%202993%202999%203015%207085%205213%205215%2034123%2034121%2034120%2034122%2041622%2034147%2034117%2034118%2034119%2034189%2034124%2034188%2032449%2032441%2034115%2034161%2034116%2032448%2032447%2032450%2032439%2032438%2032442%2032443%2032446%2032440%2032436%2032444%2032437%2032434%2023240%2032435%2023243%2023249%2023239%2023241%2023244%2023246%2023248%2032445%2023247%2023245%2023242%2023250%2039671%207464%2041594%2041592%2019194%2041591%2041593%2051132%2034100%2012319%206508%2040876%2019159%2012318%2023254%2034102%2012303%2019185%2023253%2050569%2019160%2012311%2012312%2019161%2020484%2034208%2050559%2012307%209061%2040874%2057480%2034101%2019158%2012306%2012308%2012322%2012317%2012313%2012315%2012314%209057%209058%2019188%2050564%2050565%209059%2040873%2019193%2040875%2012316%2012321%209060%2012320%2050568%2024280%2012310%2040872%2012309%2019157%2012304%2012305%2050562%2040871%2050567%2050570%2050561%2050560%2050558%2019169%2034209%2050571%2019171%2050556%2012323%2017542%2050557%2034207%2034212%2019178%2019173%2023252%2020496%2034211%2034210%2050563%209056%2034206%2019186%2019179%2019182%2034173%2019187%2050566%2034163%2019174%2034169%2019180%2034168%2034166%2019170%2034167%2034203%2019184%2034174%2019183%2034214%2034172%2034177%2034213%2034204%2019172%2019181%2034165%2019176%2034178%2034164%2019177%2034170%2034176%2034171%2034175%2019175%2034205%2068644%2015028%2032301%2032299%2015029%2032300%2048145%2048143%2048141%2048142%2048144%2048140%2041438%2041447%2041437%2041431%2041439%2041445%2041430%2041449%2041441%2041440%2048882%2041434%2041433%2041442%2041428%2041429%2041444%2041446%2041443%2066078%2041427%2041435%2048883%2041448%2041450%2017372%2013244%2013247%2012026%2013249%2013246%2017377%2012030%2012028%2017375%2020445%2057478%2059412%2012025%2057477%2013245%2019123%2017373%2013248%2012031%2017374%2012029%2057476%2017376%2013250%2051067%2051073%2051066%2051069%2041461%2041453%2041454%2041457%2041464%2041451%2041460%2041465%2041466%2041459%2041463%2041456%2061415%2041462%2041458%209826%209827%2092%209828%2046047%2046041%2046046%2046043%2046045%209829%209830%2090%2090%2090%2046042%2091%2061419%2061414%2061416%2060856%2061412%2060855%2065817%2061413%2061489%2060852%2060854%2066663%2060851%2060857%2061408%2061411%2061410%2061409%2061418%2060853%2066664%2061417%2064167%2058531%2058526%2041419%2058588%2058529%2058528%2041426%2058589%2064164%2041422%2041423%2058527%2064168%2048884%2064166%2064165%2041424%2041425%2064169%2041420%2041421%2064170%2039557%2041470%2041472%2041467%2041469%2034110%2059413%2034109%2032352%2020486%2019127%2019128%2019124%2034190%2034114%2034111%2019125%2034112%2032380%2032377%2032382%2032378%2032375%2032379%2032376%2032353%2032374%2032354%2032381%2032355%2034151%2034094%2034152%2032373%206142%2020052%206140%2044293%2066385%2032356%206797%2032363%206149%2032365%206145%2012039%2032358%2030164%2012038%206146%2032359%206147%2034224%2030165%2058240%2032370%206144%2023181%2032362%2034227%2030163%2012040%2023191%2023200%2023180%2023184%2023202%2023194%2023190%2023187%2032360%2024192%2023186%2023199%2023193%2023196%2023198%2023203%2023188%2023182%2023201%2032368%2023195%2034226%2023192%2032357%2032364%2032371%2023183%206798%2034223%2023189%2032366%2023185%206148%2023197%2012036%2012035%2032367%2032361%2012034%2034225%2032372%2032304%2034228%2032369%2063454%206151%2050003%2014538%2064446%2050005%2064082%2063702%2064079%2014547%2063440%2064222%206155%2064032%2064110%2064064%2063448%2063431%2063443%2063453%2063815%2064083%2063716%2063818%2063701%2064007%2064108%2064033%2064085%2064219%2050004%2064080%2064101%2063446%2064292%2064084%2063452%2064014%2029790%2029783%2014543%2014525%2029787%2014549%206153%2014535%2014548%2014541%2014530%2014533%2014532%2014539%2014550%2014534%2014536%2064081%206154%2014527%2014529%2029789%2014528%2064031%2014544%2049519%2014531%206152%2029791%2014542%2014540%2064454%2014526%2063703%2014546%2029785%2014545%2029788%2064109%2029784%207248%207246%207245%207244%207247%207241%2034742%2034740%2034734%2034736%2034731%2034739%2019642%2034741%2034727%2034732%2034730%2034735%2034738%2034728%2034733%2042891%2034720%2034721%2034719%2042889%2034725%2034722%2059274%2034726%2042888%2034723%2023949%206561%209686%204031%20200%2068666%2068667%2068668%2068669%2068670%2068671%2068672%2068673%2068674%2018784%2018787%2018793%2018777%2018799%2018794%2018776%2018778%2018789%2018802%2018792%2018791%2018785%2018804%2018796%2018800%2018782%2018775%2018803%2018780%2018807%2018798%2018810%2018805%2018790%2018786%2018788%2018809%2018806%2018801%2018779%2018784%2018783%202113%2044026%202112%2044027%2044028%2042356%2043980%2042355%2024765%2024759%2024761%2024758%2024762%2024764%2045103%2045104%2045102%2045093%2045100%2045101%2045105%2045094%2045098%2045107%2045097%2045106%2045108%2045096%2045095%2045109%2015530%2015531%2015528%2015529%2051071%2016278%2043294%2043297%2043291%2043290%2043293%2043292%2043295%2043289%2043287%2039849%2039850%2039848%2018984%2061377%2061376%2034150%2030180%2023216%2034149%2030179%2023214%2023215%2043944%206174%2030182%2034230%2030181%2020959%2028894%2028901%2028887%2028896%2028905%2028889%2028891%2028890%2028888%2028902%2028892%2028904%2028893%2028883%2028882%2028895%2028884%2028899%2028898%2028886%2028897%2031325%2031328%2031330%2031329%2031323%2031324%2031327%2031326%2031331%2049591%2049592%2045716%2045717%2048880%2045715%2049263%2049264%2049265%2015545%2015542%2015538%2015543%2015539%2015540%2015541%2015544%2051507%2051513%2051510%2051511%2051512%2051549%2057082%2050783%2059919%2058828%2059997%2052446%2052438%2064602%2064603%2039659%2055756%2064599%2039658%2018969%2018968%2018970%2039818%2039816%2039817%2049316%2044412%2039572%2044411%2028574%2028578%2028581%2028575%2028582%2028576%2028577%2028583%2028579%2016211%2016210%2016209%2016206%2016205%2016208%2016207%2016202%2016203%2012451%2012448%2012449%2012445%2012447%2030339%2030340%2028580%2028605%2020547%2018255%2016196%2016167%2016198%2016200%2016199%2016201%2064635%2039141%2012446%2012444%2012452%2012453%2029896%2047402%2039571%2047401%2047403%2029890%2029897%2029895%2029891%2029893%2029894%2029888%2029892%2029889%2029898%2028606%2028591%2028590%2028587%2028586%2028589%2028588%2028598%2028592%2028594%2017030%2017028%2017031%2017029%2017032%2028595%2028600%2028585%2028601%2028602%2028603%2018259%2018257%2028604%2018258%2018270%2018262%2018269%2018260%2018263%2028599%2028593%2028597%2028596%2016213%2016214%2016217%2016215%2016216%2016218%2018267%2018268%2018265%2018266%2016220%2016224%2016222%2016221%2016223%2017367%201386%205191%2017368%2017365%2017364%2017363%2017366%20445%20444%20444%20(2)%2055592%2045678%2045677%2045679%2045676%2023082%2055750%2055752%2055753%2022199%2022200%2022201%2022202%2022203%2022204%2022205%2022206%2022207%2022208%2022209%2022210%2022211%2022212%2022214%2022215%2022216%2022217%2022218%2022219%2022220%2022221%2046231%2046232%2046234%2046235%2046236%2046237%2046238%2046239%2046240%2046241%2046242%2046243%2046244%2046245%2046246%2046247%2046248%2046249%2046250%2046251%2046252%2046253%2046254%2046255%2046256%2046257%2046258%2046259%2046260%2046261%2046262%2046263%2046264%2046265%2046266%2046267%2046268%2046269%2046270%2046271%2046272%2046273%2046274%2046275%2046276%2046277%2046278%2046279%2046280%2046281%2046282%2046283%2046284%2018965%2046285%2046286%2046287%2046288%2046289%2046290%2046291%2046292%2046293%2046294%2046295%2046296%2046314%2046765%2046766%2046767%2046768%2046769%2046770%2046771%2046772%2046773%2046774%2046775%2046776%2046777%2046778%2046779%2046780%2046781%2046782%2046783%2046784%2046785%2046786%2046787%2046788%2046789%2046790%2046791%2046792%2046793%2046794%2046795%2046796%2046797%2046798%2046799%2046800%2046801%2046802%2046803%2046804%2046805%2046806%2046807%2046808%2046809%2046810%2046811%2046812%2046813%2046814%2046815%2046816%2046817%2046818%2046819%2046820%2046821%2046822%2046823%2046824%2046825%2046826%2046827%2039570%2041085%2041086%2041087%2041088%2041089%2044360%2044361%2044362%2044363%2044364%2044365%2044366%2044367%2044368%2044369%2044370%2044371%2044372%2044373%2044374%2044375%2044376%2044377%2044378%2044379%2044380%2044381%2044382%2044383%2044384%2044385%2044386%2044387%2044388%2044389%2044390%2044391%2044392%2044393%2044394%2044395%2044396%2044397%2044398%2044399%2044400%2044401%2046317%2046318%2046319%2046320%2046321%2046355%2046356%2046357%2046358%2046359%2046360%2046361%2046362%2046363%2046364%2046365%2046366%2046367%2046368%2046369%2046370%2046371%2046372%2046373%2046374%2046375%2046376%2046377%2046378%2046379%2046380%2046381%2046382%2046383%2046384%2046385%2046386%2046387%2046388%2046389%2046390%2046391%2046392%2046393%2046394%2046395%2046396%2046397%2046398%2046399%2046400%2046401%2046402%2046403%2046404%2046405%2046406%2046407%2046408%2046409%2046410%2046411%2046412%2046413%2046414%2046415%2046416%2046417%2046418%2046419%2046420%2046421%2046422%2046423%2046424%2046425%2046426%2046427%2046428%2046429%2046430%2046431%2046432%2046433%2046434%2046435%2046436%2046437%2046438%2046439%2046440%2046441%2046442%2046443%2046444%2046445%2046446%2046447%2046448%2046449%2046450%2046451%2049310%2049311%2049312%2021087%2021099%2021096%2018427%2021097%2021094%2020602%2018426%2021093%2021088%2021098%2018425%2018428%2021086%2022955%2021092%2020698%2021091%2021090%2021089%2018421%2024505%2024504%2018295%2018415%2018272%2018280%2018292%2018351%2018279%2018357%2018277%2018371%2018282%2018283%2018382%2018320%2018288%2018358%2018397%2018417%2018360%2018335%2018376%2018398%2018323%2018346%2018404%2018377%2018407%2018326%2018285%2018332%2018315%2018369%2018281%2018339%2018359%2018403%2018391%2018348%2018410%2018318%2018316%2018352%2018414%2018314%2018310%2018350%2018402%2018405%2018309%2018411%2018368%2018353%2018381%2018302%2018327%2018366%2018387%2018409%2047104%2047105%2047103%2060113%2017807%2016225%2016226%2012763%2012762%2012764%2018971%2039820%2039822%2039821%2016267%2016268%2039823%2039824%2016266%2040744%2026154%2026156%2026155%2026023%2028681%2023912%2023913%2032291%2032290%2030089%2030088%2030084%2030085%2030086%2030083%2030087%2025985%2025987%2025986%2045876%2031332%2045877%2045878%2040778%2028616%2028617%2028615%2028618%2028633%2028619%2028621%2028623%2028622%2028614%2028620%2028613%2028629%2028638%2028628%2016259%2016261%2013231%2013236%2013235%2013240%2042192%2013238%2013237%2013241%2013234%2013232%2013239%2013233%2013242%2026487%2026485%2026481%2026484%2026486%2026483%2027261%2027262%2026482%2027265%2027263%2027260%2027259%2027266%2027264%202713%202711%202715%2047965%2060644%2043328%202723%205194%2010455%202717%202719%205199%203176%202709%2010457%2010456%2010452%202712%202722%202721%202714%202708%2010454%2010453%2010458%202720%205195%202716%202710%2010459%209707%202724%209708%2018129%205196%203177%205198%202718%202707%205200%205197%2043404%2047985%2043403%2043315%2020181%2020178%2022100%2023942%2043192%2043402%2050362%2039004%202818%202819%202820%202821%202822%202823%203713%203714%205414%205415%205416%205417%205418%206024%206025%206593%206594%206595%206596%206597%206598%206599%206600%206601%206602%206603%206604%206605%206606%206607%206801%206802%206803%206804%206805%206806%206807%206808%206809%206810%2010460%2013690%2013819%2013820%2013821%2013822%2013823%2013824%2013825%2013826%2014109%2014110%2017389%2017390%2017391%2019858%2019881%2030167%2030168%2030169%2030170%2030171%2030172%2030173%2030174%2055869%2055870%2055871%2055872%2059045%2059046%2059091%2051402%2051405%2051411%2051410%2045883%2045881%2051412%2051413%2045882%2051409%2051407%2040780%2051401%2045880%2040779%2051408%2051406%2051404%2051403%2052163%2028885%2064134%2063727%2063617%2063783%2041589%2040859%2041590%2040860%2040857%2040867%2040863%2040862%2040866%2041588%2040865%2040861%2040864%2040856%2040858%2060597%2032420%2032422%2012175%2012187%2032415%2060599%2060598%2012174%2012178%2012170%2020493%2012173%2012185%2012188%2012182%2012180%2020492%2012181%2032418%2012183%2012172%2012177%2012190%2012184%2012189%2012186%2012179%2012176%2012171%2012169%2019068%2019070%2019071%2019067%2019069%2019137%2019138%2019141%2019139%2019142%2019136%2019140%2056010%2056008%2056014%2056007%2056009%2056013%2056011%2059405%2059404%2056012%2057808%2056564%2056560%2056558%2056562%2056553%2056561%2056557%2056552%2056563%2056559%2056556%2056555%2056554%2013293%206636%206637%206633%206635%206632%206634%206630%2052688%2052690%2052691%2052684%2052689%2052692%2052686%2052681%2052679%2052682%2052680%2052687%206616%2059094%206614%2059092%206618%206619%2059047%206617%206612%206615%2059093%206613%2017457%2059048%2017438%2017439%2017437%2017440%206026%2013842%2013830%206013%2013831%2014115%2013829%205470%205469%2013832%2013836%2013835%202878%2013833%2014114%2013834%2013840%2013841%2013839%2013837%2013838%207337%201308%207353%206631%207351%207342%207344%207340%207355%207338%207352%207341%207343%2013299%207354%2013297%201306%2013300%207345%2013295%2013296%207350%2013298%2013301%2013294%207347%204663%204670%204672%204661%204653%204656%204667%204671%204655%204668%204659%204664%204665%204669%204662%204673%204658%204674%204654%204660%204675%204677%204666%204678%204657%2056988%204676%2018651%207346%2018650%201307%2023933%2018652%206638%207339%201309%2024279%2040855%2057446%2019135%2057442%2057445%2057444%2057443%2025589%204695%204685%204686%204683%204697%204692%204696%204680%204699%204679%204691%204690%204681%204698%204693%204701%204682%204702%204694%204688%20448(2)%204700%204684%20448%204687%20449%204689%2017448%2017453%2017455%2017444%2017452%2017447%2017451%2017445%2017446%2017436%2017449%2017435%2017450%2017454%2043380%205029%2060536%205025%205014%205011%205013%205027%205007%205023%205022%205012%205018%205002%205008%206820%205009%201289%204997%205030%205006%205004%205028%202638%202637%202640%202639%202644%202642%202634%202628%202624%202633%202630%202627%202636%202652%202651%202648%202649%202650%202646%202632%202643%202626%202647%202635%202631%202625%202645%202629%202623%202641%203362%203352%203353%203356%203340%203346%203366%203354%203345%203351%203343%203369%203368%203359%203347%203367%203349%203357%2010619%203363%203339%203348%203365%203342%203360%2010616%203358%203355%2010618%203361%203364%203350%203344%2010617%2025266%2025252%2025255%2025265%2025249%2025268%2025263%2025254%2025267%2025251%2025259%2025256%2025264%2025262%2025260%2025257%2025243%2025261%2025258%2025241%2025235%2025221%2025247%2025240%2025244%2025245%2025228%2025253%2025246%2025242%2025237%2025236%2025248%2025232%2025231%2025207%2025223%2025230%2025226%2025227%2025234%2025233%2025211%2025225%2025224%2025210%2025250%2025214%2025239%2025238%2025209%2025205%2025215%2025213%2025212%2025229%2025222%2025217%2025218%2025216%2025220%2025206%2025208%2025219%2035403%2046066%2063754%2012032%2014537%2018781%2012033%2028443%2028442%2026966%2026976%2066449%2066457%2066480%2066423%2020055%2020066%2012129%2012145%207392%2013390%2023916%2019945%2029135%2029134%2066699%2050383%2064706%2048373%2029913%2016670%2016668%2044422%2048341%2061247%2061239%2061266%2061272%2064156%2064155%2066084%2030407%2030367%2044424%2048944%2056476%208994%208996%2041130%2050724%2020960%2020962%2038362%2038363%2038364%2048949%2016800%208963%208945%2035011%2035007%2068649%2068650%2068363%2068364%2068373%2068374%2068408%2068409%2068440%2068441%2068471%2068472%2068508%2068509%2068525%2068526%2068530%2068531%2068533%2068534%2068535%2068536%2068539%2068540%2068549%2068550%2068568%2068569%2068581%2068582%2068608%2068609%2068642%2051439%2015030%2041436%2041432%2012027%2019126%2041455%2041452%20113%20113%2065818%2066673%2058530%2048885%2041471%2041468%2020485%2034113%2034093%2034153%2023179%206143%206141%206139%2044292%2044294%2012037%2058241%2064221%2064448%2029786%2064015%2016277%2051070%207240%205954%205953%207243%207242%2019629%2019630%2034729%2034737%2052719%2025655%2042890%2034724%2023950%206560%2068675%2068676%2019779%2023899%2023900%2018795%2018808%2044029%202114%2013057%2042354%2052711%2059506%2024760%2024763%2051799%207540%207541%2036053%2036054%2045092%2045099%2015533%2015532%2051072%2051068%2043296%2043288%2028864%2028836%2039851%2044423%2028777%2028793%2028817%2028811%2018977%2049669%2018983%2052559%2049668%2049672%206291%206294%207381%207362%2016835%2043896%2023460%2023459%2060648%2060647%209439%2018136%205438%205434%2018601%2018603%2030420%2066404%2039142%2048881%2028881%2028900%2039669%2043376%206276%207336%207366%207371&sp=UCM%20test

**Comment by ebraker** [comment]
Ack! Didn't realize my download doesn't reflect my customized fields. Now I am getting the same 431 when trying to request data.

**Comment by dustymc** [comment]
Yep, that'll do it. I have no control over that limitation, and don't/can't-ish use POST there so URLs can be shared.

>Didn't realize my download doesn't reflect my customized fields

Hu?

**Comment by ebraker** [comment]
Ok some weird thing was just happening but just cleared after ~30 minutes. The profile I recently created was showing at the top of the screen: 
![image](https://github.com/ArctosDB/arctos/assets/11336485/0b5024a6-cd09-4793-a54f-371b643f33b1)

But none of the results fields I had customized were visible - I have 24 fields selected but was just getting this handful and therefore my download didn't reflect my customization:
![image](https://github.com/ArctosDB/arctos/assets/11336485/d8cb623e-5956-43e6-aa94-d80f1742251a)

It just refreshed and worked.

However, I still get the 431 error when I select "request data" when using the long URL query. I think I can cobble together what I need between part table view and download and lookups.
